xclim/indicators/atmos/_precip.py

```diff
@@ -291,17 +294,58 @@ class HrPrecip(Hourly):
 )
 
 
-fire_weather_indexes = FireWeather(
-    identifier="fwi",
+def fire_weather_indexes(*args, **kwargs):
+
+    warnings.warn(f"`fire_weather_indexes` indicator is deprecated as of xclim v0.37.18-beta. This indicator "
+                  f"has been renamed to `cffwis_indices` to better support international collaboration. "
+                  f"The `fire_weather_indexes` alias will be removed in xclim v0.39."
+                  f" See the xclim release notes for more information.", DeprecationWarning, stacklevel=2)
+
+    return FireWeather(
+        identifier="fire",
+        realm="atmos",
+        var_name=["dc", "dmc", "ffmc", "isi", "bui", "fire"],
+        standard_name=[
+            "drought_code",
+            "duff_moisture_code",
+            "fine_fuel_moisture_code",
+            "initial_spread_index",
+            "buildup_index",
+            "fire_weather_index",
+        ],
+        long_name=[
+            "Drought Code",
+            "Duff Moisture Code",
+            "Fine Fuel Moisture Code",
+            "Initial Spread Index",
+            "Buildup Index",
+            "Fire Weather Index",
+        ],
+        description=[
+            "Numeric rating of the average moisture content of deep, compact organic layers.",
+            "Numeric rating of the average moisture content of loosely compacted organic layers of moderate depth.",
+            "Numeric rating of the average moisture content of litter and other cured fine fuels.",
+            "Numeric rating of the expected rate of fire spread.",
+            "Numeric rating of the total amount of fuel available for combustion.",
+            "Numeric rating of fire intensity.",
+        ],
+        units="",
+        compute=indices.cffwis_indices,
+        missing="skip",
+        )(*args, **kwargs)
+
+
+cffwis_indices = FireWeather(
+    identifier="fire",
```

Comment:
```suggestion
    identifier="cffwis",
```
Same as Travis. The variable name has to stay "fwi" and the word "fire" is too broad. 
Haha I suspect a find-and-replace? ;)

Reply:
When changing the `identifier`, ensure the references to it are also modified! I'll point them.

Reply:
Yeah, I realized that my find and replace was a bit aggressive. Reverting.